src/components/PasswordResetForm.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { Form, Button } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ButtonSpinner from './ButtonSpinner';
+import { passwordReset } from '../store/actions/passwordResetActions';
+import validate from '../utils/registerFormValidator';
+import store from '../store/store';
+import { IS_LOADING } from '../store/actions/actionTypes';
+
+export class ConnectedPasswordResetForm extends Component {
+  constructor() {
+    super();
+    this.state = {
+      confirmPassword: '',
+      password: '',
+      errors: {},
+    };
+  }
+
+  handleChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleSubmit = (event) => {
+    const { password, confirmPassword } = this.state;
+    const { PasswordReset } = this.props;
+    event.preventDefault();
+    const errors = validate('', '', password, confirmPassword);
+    const token = localStorage.resetToken;
```

Comment:
@ElMonstro, it would be best to pick the token from `window.location`

Reply:
> @ElMonstro, it would be best to pick the token from `window.location`
I can't do that with my current architecture

Reply:
What do you mean, I believe it is possible.